### PR TITLE
terminology for ebml spec

### PR DESCRIPTION
**Pull request by dericed**
Adding this based on a comment from Tim’s review. Some questions/comments:
- should we add more or remove some of these?
- this increases redundancy in the overall spec, so some sentences may be removed from later in the doc, but I’d like to finalize this section first before doing the trimming.
- should we backtick all defined terms? (lots of backticking)

### CODE REVIEW

**Comment by robUx4** [comment]
As you mention it's a WIP. Do you want this merged or you will continue working on this branch ?

**Comment by dericed** [comment]
Was hoping to see some discussion on the questions posed above first, but no objection to merging it now if it looks okay to you.

**Comment by mbunkus** [comment]
@dericed I'm a bit busy at the moment. I've read all the issues and pull requests you've opened over the last couple of days, but it'll probably take me until the weekend to respond to them. You may rest assured that I haven't forgotten about them :)

**Comment by mbunkus** [comment]
@dericed I'm a bit busy at the moment. I've read all the issues and pull requests you've opened over the last couple of days, but it'll probably take me until the weekend to respond to them. You may rest assured that I haven't forgotten about them :)

**Comment by dericed** [comment]
@mbunkus no worries and thanks. This PR is relatively small. Within the https://github.com/Matroska-Org/ebml-specification/commits/clarifications branch I'm working on commits based on Tim's comments at https://mailarchive.ietf.org/arch/msg/cellar/Mfj44tK1gyfqjU3Uu1tI5fMovtk which is a comprehensive set of suggestions and comments to work through, though I should be done and have the PR ready before the weekend. From Tim's email I'm either responding with an issue or a commit but most of it should be worked through and in a PR soon.

**Comment by mbunkus** [comment]
> should we add more or remove some of these?
> this increases redundancy in the overall spec, so some sentences may be removed from later in the doc, but I’d like to finalize this section first before doing the trimming.

I'm fine with this PR; I don't mind the redundancy.

> should we backtick all defined terms? (lots of backticking)

Either backtick all of them or none. Anything else is prone to cause confusion. My vote is therefore: yes, do backtick everything.

**Comment by dericed** [comment]
I suggest waiting until after https://github.com/Matroska-Org/ebml-specification/pull/99 is resolved to backtick everywhere to avoid a huge conflict. Filed this issue to remember: https://github.com/Matroska-Org/ebml-specification/issues/100.

File: specification.markdown [hunk 1]
```diff
@@ -12,7 +12,58 @@ EBML uses a simple approach of building Elements upon three pieces of data (tag,
...
+`Empty Element`: An `Empty Element` is an `EBML Element` that has an `Element Data Size` with all VINT\_DATA bits set to zero which indicates that the `Element Data` of the Element is zero octets in length.
+
+`Level`: `Level` notes at what hierarchical depth the `EBML Element` may occur within an `EBML Document`, starting from zero.
+
+`Master Element`: The `Master Element` contains zero, one, or many other `EBML Elements`.
+
+`Parent Element`: A relative term to describe the `Master Element` which contains a specified element.
+
+`Root Element`: A mandatory, non-repeating EBML Element which occurs within an `EBML Document` at `Level 0` and contains all other `EBML Elements` of the `EBML 
+Document`, excepting the `EBML Header` and optional Void Elements.
```
Comment (retokromer) [inline specification.markdown]: delete newline between EBML and Document

File: specification.markdown [hunk 2]
```diff
@@ -12,7 +12,58 @@ EBML uses a simple approach of building Elements upon three pieces of data (tag,
...
+
+`Top-Level Element`: An elements defined to only occur at `Level 1` within the `Root Element`.
+
+`Unknown-Sized Element`: An Element with an unknown `Element Data Size`.
```
Comment (retokromer) [inline specification.markdown]: either "An element" here (I guess this is meant) or "An Element" in the previous one.

**Comment by dericed** [comment]
Fixed issues per @retokromer's comments in https://github.com/Matroska-Org/ebml-specification/pull/87/commits/2b3906490b2202e30005dbb7ff4ba0816fb20fd3. Ready for re-review.

**Pull request merged**